### ISSUE TITLE
Production build does not resolve tsconfig.json paths.

### ISSUE THREAD

**Issue by infacto** [infacto/forum#1]
Hi @ngfk 

sorry for disturbing you. It's about a 3 years old issue. 😃 
I'm still on Ionic 3 with `@ionic/app-scripts: 3.1.11`. Can't update our huge app to Ionic 4+, anyway.
I have the same problem you descripted on the achieved Ionic app scripts repo.
With this issue I want to continue the conversation. You posted a [solution](https://github.com/ionic-team/ionic-app-scripts/issues/1126#issuecomment-333090133). But I have no idea who to work with this. Can you help me please? 
I thought I cannot access the webpack config of Ionic 3.

What is you file structure? 
You created just a file **webpack.config.js** in the root directory of your app. With the content:

```js
const { join } = require('path');
const webpackMerge = require('webpack-merge');
const { dev, prod } = require('@ionic/app-scripts/config/webpack.config');

const customConfig = {
  resolve: {
    alias: {
      app: join(__dirname, './src'),
    }
  }
};

module.exports = {
  dev: webpackMerge(dev, customConfig),
  prod: webpackMerge(prod, customConfig),
}
```

From issue: https://github.com/ionic-team/ionic-app-scripts/issues/1126

**Comment by ngfk** [comment]
Hey @infacto, 

I no longer use Ionic 3, so it's no familiar terrain for me anymore. And I also switched jobs so I can't check the file structure.

But from the reproduction steps I see that I also had to configure Ionic to use the custom **webpack.config.js** file. Did you also perform this step?

* Tell ionic to use our custom config. Add to **package.json**:
```
"config": {
  "ionic_bundler": "webpack",
  "ionic_webpack": "./webpack.config.js"
}
```
